### PR TITLE
#3965 - Samples for even participants are not shown - hot fix

### PR DESCRIPTION
**Pull request by sergiupacurariu**

### CODE REVIEW

File: sormas-backend/src/main/java/de/symeda/sormas/backend/sample/SampleFacadeEjb.java [hunk 1]
```diff
@@ -809,7 +812,13 @@ private void pseudonymizeDto(Sample source, SampleDto dto, Pseudonymizer pseudon
 			boolean isInJurisdiction = sampleJurisdictionChecker.isInJurisdictionOrOwned(sampleJurisdiction);
 			User currentUser = userService.getCurrentUser();
 
-			pseudonymizer.pseudonymizeDto(SampleDto.class, dto, isInJurisdiction, s -> {
+			boolean samplePseudonimized = true;
```
Comment (MateStrysewske) [inline sormas-backend/src/main/java/de/symeda/sormas/backend/sample/SampleFacadeEjb.java:815]: => samplePseudonymized

**Review by MateStrysewske — CHANGES_REQUESTED** [review]
Check code formatting :)

File: sormas-backend/src/main/java/de/symeda/sormas/backend/sample/SampleFacadeEjb.java (hunk 1 above)
Comment (sergiupacurariu) [reply sormas-backend/src/main/java/de/symeda/sormas/backend/sample/SampleFacadeEjb.java:815]: done

**Comment by sergiupacurariu** [comment]
code formatting was updated

**Pull request closed without merging**